<ui version="4.0" >
 <class>SkanliteSettings</class>
 <widget class="QWidget" name="SkanliteSettings" >
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QCheckBox" name="showB4Save" >
     <property name="text" >
      <string>Preview the image before saving.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Save mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QComboBox" name="saveModeCB" >
     <item>
      <property name="text" >
       <string>Open the save dialog for every image</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Open the save dialog for the first image only</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Save without asking for a filename</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Save Location:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>0</number>
     </property>
     <property name="topMargin" >
      <number>0</number>
     </property>
     <property name="rightMargin" >
      <number>0</number>
     </property>
     <property name="bottomMargin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="saveDirLEdit" />
     </item>
     <item>
      <widget class="QPushButton" name="getDirButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Name &amp; Format:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLineEdit" name="imgPrefix" >
       <property name="text" >
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>###.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="imgFormat" />
     </item>
    </layout>
   </item>
   <item row="3" column="2" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <widget class="QCheckBox" name="setQuality" >
     <property name="text" >
      <string>Specify quality:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QFrame" name="imgQualityFrame" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QSlider" name="horizontalSlider" >
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>85</number>
        </property>
        <property name="sliderPosition" >
         <number>85</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="imgQuality" >
        <property name="suffix" >
         <string>%</string>
        </property>
        <property name="maximum" >
         <number>100</number>
        </property>
        <property name="value" >
         <number>85</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>setQuality</sender>
   <signal>toggled(bool)</signal>
   <receiver>imgQualityFrame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>45</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel" >
     <x>385</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>imgQuality</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>323</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel" >
     <x>414</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imgQuality</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>422</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel" >
     <x>254</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
